code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -82,7 +77,6 @@
 /atom/movable/screen/plane_master/game_world_fov_hidden
 	name = "game world fov hidden plane master"
 	plane = GAME_PLANE_FOV_HIDDEN
-	render_relay_plane = GAME_PLANE
```

Comment:
GAME_PLANE is defined as -4, setting this to inherit the value of RENDER_PLANE_GAME changes it to 990.

Is this a fix and what does this change?

In addition, `/atom/movable/screen/plane_master/game_world_above` also has `render_relay_plane = GAME_PLANE` but is unchanged?

Reply:
typo